projects/ngrx.io/src/app/custom-elements/events/event-order-by.pipe.ts

```diff
@@ -0,0 +1,25 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Event } from './event.model';
+
+type EventOrderBy = 'earlierToLater' | 'laterToEarlier';
```

Comment:
What about just 'ascending' and `descending'?

Reply:
Fixed in 26275a5.